src/resolvers/block.ts

```diff
@@ -23,7 +23,7 @@ const block: IFieldResolver<any, any> = async (obj, args: IBlockArgs) => {
     throw new Error('Only one of number, hash or tag argument should be provided.');
   }
 
-  const block = await web3.eth.getBlock(params[0]);
+  const block = await web3.eth.getBlock(params[0], true);
```

Comment:
What was the reason for this change?

Reply:
When false (default) getBlock only returns a block with transaction hashes but when true it returns a block with the transaction objects, which are needed by transactionAt.

Reply:
LGTM - Raul will review and approve :)

Reply:
Looks good, yeah.